app/sose2019/views.py

```diff
@@ -162,21 +162,21 @@ def __init__(self, **kwargs):
 #       ])
     tshirt = SelectField('T-Shirt', choices = T_SHIRT_CHOICES)
     addtshirt = IntegerField('Anzahl zusätzliche T-Shirts',[validators.optional()], widget=NumberInput())
-    bottle = BooleanField('Ich möchte eine Wasserflasche bestellen (ca 10€, verbindliche Kaufzusage)')
+    bottle = BooleanField('Ich möchte eine Wasserflasche bestellen (ca 10€, verbindliche Kaufzusage) Wer mehr will, schreibt das in das Kommentarfeld')
     quant = IntegerField('Ich möchte Quanten-Aufkleber (2€ pro 40er Bogen, verbindliche Kaufzusage)', [validators.optional()], widget=NumberInput())
 #   hoodie = SelectField('Hoodie', choices = HOODIE_CHOICES)
 #   handtuch = BooleanField('Ich möchte gerne ein Handtuch bestellen')
 #   roemer = BooleanField('Ich möchte gerne einen Weinrömer bestellen')
-    krug = BooleanField('Ich möchte einen Bierkrug bestellen (ca 5€, verbindliche Kaufzusage)')
+    krug = BooleanField('Ich möchte einen Bierkrug (0,4l) bestellen (ca 5€, verbindliche Kaufzusage) Wer mehr will, schreibt das in das Kommentarfeld')
     bierak = BooleanField('Ich möchte am Bieraustausch AK teilnehmen (5€ Teilnahmegebühr)(neuer Modus, Bier wird gestellt, siehe E-Mail)')
-    zaepfchen = SelectField('Bist du ein Zäpfchen?', choices=[
+    zaepfchen = SelectField('komsmt du zum ersten mal zu einer ZaPF?', choices=[
         ('ja','Ja'),
         ('jaund','Ja und ich hätte gerne einen ZaPF-Mentor.'),
         ('nein','Nein'),
         ])
     mentor = BooleanField('Ich möchte ZaPF-Mentor werden und erkläre mich damit einverstanden, dass meine E-Mail-Adresse an ein Zäpfchen weitergegeben wird.')
     foto = BooleanField('Ich bin damit einverstanden, dass Fotos von mir gemacht werden.')
-    halle = BooleanField('Ich habe die Hallenordnung gelesen und verstanden und werde mich daran halten.', [validators.InputRequired()])
+    halle = BooleanField('Ich habe die Hallenordnung (siehe Website) gelesen und verstanden und werde mich daran halten.', [validators.InputRequired()])
```

Comment:
Ihr solltet hier Links machen können... `<a href="zapf.in">Website</a>`

Reply:
Eingefügt.